cg/meta/workflow/rnafusion.py

```diff
@@ -324,3 +327,48 @@ def report_deliver(self, case_id: str) -> None:
                 )
             )
         )
+
+    def get_multiqc_json_path(self, case_id: str) -> Path:
+        """Genereates a path where the multiqc_data.json file should be located."""
+        return Path(self.root_dir, case_id, "multiqc", "multiqc_data", "multiqc_data.json")
+
+    def get_metrics_deliverables_path(self, case_id: str) -> Path:
+        """Genereates a path where the <case>_metrics_deliverables.yaml file should be located."""
+        return Path(self.root_dir, case_id, f"{case_id}_metrics_deliverables.yaml")
+
+    def parse_multiqc_json(self, case_id: str) -> List[Dict]:
+        """Parses a multiqc_data.json file and returns metrics and values formatted."""
+        multiqc_json = MultiqcDataJson(
+            **read_json(file_path=self.get_multiqc_json_path(case_id=case_id))
+        )
+        metrics_values = {}
+        for section in multiqc_json.report_general_stats_data:
+            if case_id in section:
+                metrics_values.update(list(section.values())[0])
+        return [
+            {
+                "header": None,
+                "id": case_id,
+                "input": "multiqc_data.json",
+                "name": metric_name,
+                "step": "multiqc",
+                "value": metric_value,
+                "condition": RNAFUSION_METRIC_CONDITIONS.get(metric_name, None),
+            }
+            for metric_name, metric_value in metrics_values.items()
+        ]
+
+    def write_metrics_deliverables(self, case_id: str, dry_run: bool = False):
+        """Write <case>_metrics_deliverables.yaml file."""
+        deliverables_path = self.get_metrics_deliverables_path(case_id=case_id)
+        if dry_run:
+            LOG.info(
+                f"Dry run: metrics deliverables file would be written to {deliverables_path.as_posix()}"
+            )
+            return
+        LOG.info(f"Writing metrics deliverables file to {deliverables_path.as_posix()}")
+        WriteFile.write_file_from_content(
+            content={"metrics": self.parse_multiqc_json(case_id=case_id)},
+            file_format=FileFormat.YAML,
+            file_path=deliverables_path,
+        )
```

Comment:
Should we store it in HK after this step? Or are you planning to do it in another PR?

Reply:
yes, planning to do it in a different PR

Reply:
👍